### PR TITLE
Completed CSS Brochure Form - Amelia Talkington

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-01-20 at 6 40 21 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+body {
+  align-content: center;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:3]: You can remove this.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+fieldset {
+  width: 40%;
+  border: none;
+  margin: auto;
+}
+
+h2 {
+  text-align: center;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: medium;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:14]: Stick with `rem` for font sizes, if possible.

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  color: rgb(236, 95, 89);
+  font-weight: 500;
+  letter-spacing: 1px;
+}
+
+label:not(:last-child) {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: x-small;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:22]: Stick with `rem` for font sizes, if possible.

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  border-radius: 5px;
+  border: rgb(177, 180, 179) solid 1px;
+  margin-top: 2px;
+  margin-bottom: 10px;
+}
+
+#company-email,
+#name,
+#where {
+  padding-left: 2%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:42]: Stick with `px` or `rem` for padding, if possible

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  height: 32px;
+  background-color: rgb(236, 95, 89);
+  color: white;
+  font-family: 'Brandon Grotesque', sans-serif;
+  text-transform: uppercase;
+  float: right;
+  border-radius: 20px;
+  font-weight: 500;
+  border: none;
+  font-size: small;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:64]: Stick with `rem` for font sizes, if possible

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Amelia! Please see my notes:

**Comment by ameliatalkington** [comment]
Screen Shot 2021-01-21 at 2 02 37 PM
I made the recommended changes

**Review by codingcodymiller — APPROVED** [review]
🙌  🙌  🙌

**Pull request merged**